data/xp-pen-ack05-remote.tablet

```diff
@@ -0,0 +1,39 @@
+# XP-Pen
+# ACK05 Remote
+#
+# Reports as Hanvon Ugee Shortcut Remote, for recordings
+# etc. see https://gitlab.freedesktop.org/libevdev/udev-hid-bpf/-/issues/32
+#
+#  ┌────────┐────────────────────────┐
+# /  ┌────┐  \   ┌───┐┌───┐┌───┐┌───┐│
+# │  │ A  │  │   │ B ││ C ││ D ││   ││
+# │  └────┘  │   └───┘└───┘└───┘│ H ││
+# \_________/    ┌───┐┌───┐┌───┐│   ││
+#  │             │ E ││ F ││ G ││   ││
+#  │             └───┘└───┘└───┘└───┘│
+#  │             ┌───┐┌────────┐┌───┐│
+#  │             │ I ││    J   ││ K ││
+#  │             └───┘└────────┘└───┘│
+#  └─────────────────────────────────┘
+
+[Device]
+Name=Hanvon Ugee Shortcut Remote
+ModelName=
+DeviceMatch=usb|28bd|0202
+Layout=xp-pen-ack05-remote.svg
+Class=Remote
+
+[Features]
+Stylus=false
+NumDials=1
+NumRings=0
+NumStrips=0
+StatusLEDs=Ring
```

Comment:
If the LED doesn't light up on the device, we don't really have a status led, isn't it? Or am I misunderstanding something here?